build_rules.gradle

```diff
@@ -109,7 +109,7 @@ offlineDependencies {
 
 // A class defining the set of configurable properties accepted by applyJavaNature
 class JavaNatureConfiguration {
-  double javaVersion = 1.7        // Controls the JDK source language and target compatibility
+  double javaVersion = 1.8        // Controls the JDK source language and target compatibility
```

Comment:
Can you update the build.gradle files which are passing in 1.8 right now and remove it in like `runners/gearpump/build.gradle`. 

https://github.com/apache/beam/search?utf8=%E2%9C%93&q=%22javaVersion%3A+1.8%22&type=

Reply:
Done, thanks !

Reply:
It seems you missed `sdks/java/harness/build.gradle`

Other then that, the gradle changes LGTM.

Reply:
Good catch, thanks Luke, I'm fixing that one.

Reply:
Done